camera:
  intrinsics:
    fx: 1037.286354444401
    fy: 1043.4288463759349
    cx: 946.0673972084953
    cy: 490.2340677407784
    distortion_coeffs: [-0.2766049460625107, 0.06972195251711784, 0.00018442181342890202, -0.0009009324579596044, 0.0]
  extrinsics:
    rotation: [1.0, 0.0, 0.0,
               0.0, 1.0, 0.0,
               0.0, 0.0, 1.0]
    translation: [0.0, 0.0, 0.45]